ddpui/api/pipeline_api.py

```diff
@@ -712,19 +712,14 @@ def get_prefect_flow_runs_log_history(
 )
 @has_permission(["can_view_pipeline"])
 def get_prefect_flow_runs_log_history_v1(
-    request, deployment_id, limit: int = 0, fetchlogs=True, offset: int = 0
+    request, deployment_id, limit: int = 0, offset: int = 0
 ):
     # pylint: disable=unused-argument
     """Fetch all flow runs for the deployment and the logs for each flow run"""
     flow_runs = prefect_service.get_flow_runs_by_deployment_id_v1(
         deployment_id=deployment_id, limit=limit, offset=offset
     )
 
-    if fetchlogs:
-        for flow_run in flow_runs:
-            logs_dict = prefect_service.get_flow_run_logs_v2(flow_run["id"])
-            flow_run["runs"] = logs_dict
-
     return flow_runs
```

Comment:
Why did we remove this ? 
The `get_flow_run_logs_v2` is not just fetching the logs its also, fetching the graph of a `flow_run_id`. We need that to display the all the tasks inside the flow_run. That is why i asked you to create a new api on proxy to just fetch the and remove logs from it. 
I agree the name of the function is confusing a bit because first it used to only fetch logs but now its get the graph (or list of tasks inside a flow run also). 

<img width="740" alt="Screenshot 2024-08-28 at 11 32 59" src="https://github.com/user-attachments/assets/f4b4eb17-c54d-47d2-ba4d-3e00c34ebe38">